requirements.txt

```diff
@@ -1,7 +1,13 @@
+# sha256: TIMmzrNyf0_3AR1_Ybe8oVzKGU4iiAE6c9XMr1bToYs
```

Comment:
We should also add the hash for the wheel (or sdist) here; there's a wheel and an sdist available, and which one is chosen is a matter of pip version and flags.

Btw, many applause on the work you've done while I'm away. Looks like DXR is in good hands!